src/Notifier.js

```diff
@@ -98,8 +105,10 @@ const Notifier = {
         this.boundOnRoomTimeline = this.onRoomTimeline.bind(this);
         this.boundOnSyncStateChange = this.onSyncStateChange.bind(this);
         this.boundOnRoomReceipt = this.onRoomReceipt.bind(this);
+        this.boundOnEventDecrypted = this.onEventDecrypted.bind(this);
         MatrixClientPeg.get().on('Room.timeline', this.boundOnRoomTimeline);
         MatrixClientPeg.get().on('Room.receipt', this.boundOnRoomReceipt);
+        MatrixClientPeg.get().on('Event.decrypted', this.boundOnEventDecrypted);
```

Comment:
I'm just not keen on having to iterate the list every time an event is decrypted...